### PR TITLE
CRS-30 feat: add change password feature

### PR DESCRIPTION
**Pull request by Goorsky1**

### CODE REVIEW

**Review by woj-mal09 — APPROVED** [review]
looks good to me, but i am kinda blind... soo...

File: backend/src/controllers/users.py [hunk 1]
```diff
@@ -11,3 +13,28 @@
...
+@users.route('/change-password', methods=['PUT'])
+@login_required
+def change_password():
+    user = User.query.get(int(session['_user_id']))
+    if user is None:
+        abort(404, description='User does not exist')
+
+    body = request.json
+    current_password, new_password, confirm_new_password = body['currentPassword'], body['newPassword'], body['confirmNewPassword']
+    validate_password_change(user,current_password,new_password, confirm_new_password)
```
Comment (kjez21) [inline backend/src/controllers/users.py:35]: Sformatuj kod plz

File: backend/src/controllers/users.py [hunk 2]
```diff
@@ -11,3 +13,28 @@
...
+    
+    user.password=bcrypt.generate_password_hash(new_password).decode("utf-8")
```
Comment (kjez21) [inline backend/src/controllers/users.py:37]: tu też

**Review by kjez21 — CHANGES_REQUESTED** [review]

File: backend/src/controllers/users.py (hunk 1 above)
Comment (Goorsky1) [reply backend/src/controllers/users.py]: k

File: backend/src/controllers/users.py (hunk 2 above)
Comment (Goorsky1) [reply backend/src/controllers/users.py]: k

**Review by kjez21 — APPROVED** [review]

**Pull request merged**